### DISCUSSION TITLE
How to add "this" node

### DISCUSSION THREAD

**Discussion by linonetwo** [FlydeHQ/flyde#39] [Q&A] [answered]
For example, in https://www.flyde.dev/playground/fibonacci/ , how to add another Fibonacci node using the UI? (not editing the yaml file)

Also, how to add `jsx` output node in the https://www.flyde.dev/playground/react-counter/ example? @GabiGrin

**Comment by GabiGrin** [comment, marked as answer]
"this" node - In the VSCode extension, it just works - you can add the node you're working on. The online playground requires a small tweak to make it work - opened #41 

`jsx` output - in that example, I simply created a new output (right click board -> new output) and then connected that output to a React renderer. See https://github.com/FlydeHQ/flyde/blob/main/website/src/pages/playground/react-counter.tsx#L32, https://github.com/FlydeHQ/flyde/blob/main/website/src/pages/playground/_OutputLogs/OutputJsx.tsx

PS: That example is merely a showcase for Flyde's flexibility, but I strongly think frontend components is not a good use-case for Flyde.

**Reply by linonetwo** [reply]
Thanks, that makes sense. Even if I want to create a UI element (for example, recommended reply buttons), I would create a node to update the redux store, so react side renders rebuild element based on state, instead of building elements on the fly.

I will inform you when I get progress on this.

**Reply by GabiGrin** [reply]
Looking forward to it!
